core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockGoogleServicesDAO.scala

```diff
@@ -223,4 +223,21 @@ class MockGoogleServicesDAO(groupsPrefix: String,
     Future.successful(billingAccount == accessibleBillingAccountName)
   }
 
+  override def getRegionForRegionalBucket(bucketName: String): Future[Option[String]] = {
+    Future.successful {
+      bucketName match {
+        case "fc-regional-bucket" => Option("europe-north1")
```

Comment:
Just FWIW I think the real API returns this as `EUROPE-NORTH1`

Reply:
That is correct. But in the actual implementation we convert it to lowercase before returning it. The reason we convert it to lowercase is that `.regions().get(..)` in [here](https://github.com/broadinstitute/rawls/blob/889918b9e49916d7ac9832539854f89b4b033e3f/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala#L425-L426) expects the input to be in lowercase. If it is in any other format it throws an error